tests/conftest.py

```diff
@@ -59,14 +63,26 @@
                 YoloV6Metric,
                 ["images"],
                 "onnx",
+                "",
+            ),
+            (
+                service,
+                "resnet18",
+                "s3://luxonis-test-bucket/modelconverter-test/imagenette-extra-small.zip",
+                ResnetMetric,
+                ["input.1"],
+                "onnx",
+                "rvc4.disable_calibration true rvc3.disable_calibration true",
             ),
         ]
         for service in [
             "rvc2",
             "rvc2_superblob",
             "rvc3",
             "rvc3_quant",
+            "rvc3_non_quant",
```

Comment:
Is "rvc3" quant or non-quant?

Reply:
"rvc3" is normal conversion but tested on the non-quant model. "rvc3_non_quant" is where calibration is explicitly disabled.